lib/watcher.js

```diff
@@ -61,6 +61,8 @@ function Watcher(logger, api, files, sources) {
 		this.busy = api.run(specificFiles || files, {
 			runOnlyExclusive: runOnlyExclusive
 		}).then(function (runStatus) {
+			runStatus.testRunElapsed = runStatus.testRunStart && process.hrtime(runStatus.testRunStart);
```

Comment:
I think @sindresorhus is saying that `Date.now()` is accurate enough. I think that's true for this use case, no point optimizing test runs down to the nanosecond, and there is more cognitive overhead with `hrtime`.

That said I don't think there's much value in tracking test duration when in watch mode, especially since the number of test files run can vary greatly.